recipes/dfviz/meta.yaml

```
@@ -0,0 +1,42 @@
+{% set version = "0.0.3" %}
+{% set name = "dfviz" %}
+{% set sha256 = "f8fd80ea7114ba71e575104b5648e0e52e0a677f8c11162d2ee1100c51898d42" %}
+
+package:
+   name:  {{ name }}
+   version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  noarch: python
+  script: "{{ PYTHON }}  -m pip install . --no-deps --ignore-installed -vv"
```

The arguments are no longer required by supplied by default.
```suggestion
  script: "{{ PYTHON }}  -m pip install . -vv"
```